bitnami/fluentd/values-production.yaml

```diff
@@ -187,6 +196,14 @@ aggregator:
   ##
   replicaCount: 2
 
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 1001
+    fsGroup: 1001
```

Comment:
in the `values.yaml` this is configured to have uid `0`, while in the `values-production,yaml` it's `1001`. Was it intended to be this way?

Reply:
It was not intended. Both should be using `0`.